packages/phaserx/package.json

```diff
@@ -7,48 +7,33 @@
     "directory": "packages/phaserx"
   },
   "license": "MIT",
-  "main": "src/index.ts",
-  "source": "src/index.ts",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "src/index.ts",
   "scripts": {
     "build": "pnpm run build:js",
-    "build:js": "rollup -c rollup.config.js",
+    "build:js": "tsup",
     "clean": "pnpm run clean:js",
     "clean:js": "rimraf dist",
+    "dev": "tsup --watch",
     "docs": "rimraf API && typedoc src && find API -type f -name '*.md' -exec sed -E -i \"\" \"s/(#.*)(<.*>)/\\1/\" {} \\;",
     "lint": "eslint . --ext .ts",
-    "prepack": "mv package.json package.json.bak && jq \".main = \\\"dist/index.js\\\"\" package.json.bak > package.json ",
-    "postpack": "mv package.json.bak package.json || echo 'no package.json.bak'",
     "release": "npm publish --access=public",
-    "test": "tsc --noEmit && jest --passWithNoTests"
+    "test": "tsc --noEmit"
   },
   "dependencies": {
-    "@use-gesture/vanilla": "10.2.9"
-  },
-  "devDependencies": {
     "@latticexyz/utils": "workspace:*",
-    "@rollup/plugin-commonjs": "^21.0.2",
-    "@rollup/plugin-node-resolve": "^13.1.3",
-    "@rollup/plugin-typescript": "^11.1.0",
-    "@types/jest": "^27.4.1",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
-    "mobx": "^6.7.0",
-    "phaser": "3.60.0-beta.14",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.70.0",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rxjs": "^7.5.5",
-    "ts-jest": "^29.0.5",
-    "tslib": "^2.5.0",
-    "typedoc": "0.23.21",
-    "typedoc-plugin-markdown": "^3.13.6",
-    "typescript": "^4.9.5"
```

Comment:
everything seems to work without `typescript` as a dev dependency, but I'm confused why (would have expected eg. `tsc` to require `typescript` as a dep)

Reply:
typescript is installed at the root (since we need it ~everywhere and its version consistent across packages)